homeassistant/helpers/system_info.py

```diff
@@ -9,6 +10,8 @@
 
 from .typing import HomeAssistantType
 
+DATE_STR_FORMAT = "%Y-%m-%d %H:%M:%S"
```

Comment:
I really do not know where it can be obtained in HA but, shouldn't the timeformat of the user's locale be used here?

Reply:
Good call out. Looks like %c might be what we could use here without pulling in Babel or some other library.

%c : Locale’s appropriate date and time representation. : 
Examples:
Tue Aug 16 21:30:00 1988 (en_US);
Di 16 Aug 21:30:00 1988 (de_DE)